weaveworks/weave-gitops-enterprise
# handle scenario where wge is already bootstrapped

Closes https://github.com/weaveworks/weave-gitops-enterprise/issues/3496

**What changed?**

- Update Install_WGE step to check if WGE is already installed, if an installation found it should skip and continue to the next steps.

**Why was this change made?**

- to avoid unnecessary work to the user if he already has wge bootstrapped 

**How was this change implemented?**

- Update the WGE installation step.

**How did you validate the change?**


- run the `gitops bootstrap` when you already have wge bootstrapped
- run the `gitops bootstrap` when you already with no wge bootstrapped

       there tests that fail _without_ the change?

**Release notes**



**Documentation Changes**



**Other follow ups**
